### DISCUSSION TITLE
Pages not showing buttons but local host does

### DISCUSSION THREAD

**Discussion by JamieBursey** [community/community#53481] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Hello, I have a project that I am trying to build but when I post to github pages some things are missing that are showing on my localhost. Anyone have an idea why this is? The Buttons seem to disappear on github pages

https://github.com/JamieBursey/Hockeysales

**Comment by fefojedani** [comment]
> ### Select Topic Area
> General
> 
> ### Body
> Hello, I have a project that I am trying to build but when I post to github pages some things are missing that are showing on my localhost. Anyone have an idea why this is? The Buttons seem to disappear on github pages
> 
> https://github.com/JamieBursey/Hockeysales
